src/interface/viewport.c

```diff
@@ -1140,6 +1097,51 @@ int sub_98198C(
 }
 
 /**
+ * rct2: 68818E
+ *
+ * @param image_id (ebx)
+ * @param x (ax)
+ * @param y (cx)
+ * @param[out] paint (ebp)
+ * @return (!CF) success
+ */
+bool sub_68818E(uint32 image_id, uint8 x, uint8 y, paint_struct ** paint)
+{
+
+	//Not a paint struct but something similar
```

Comment:
I don't think my note is really correct anymore. It definitely is a paint struct.